html/pfappserver/root/static.alt/src/components/pfFormDatetime.vue

```diff
@@ -17,7 +17,8 @@
  */
  <template>
   <b-form-group horizontal :label-cols="(columnLabel) ? labelCols : 0" :label="$t(columnLabel)" 
-    :state="isValid()" :invalid-feedback="getInvalidFeedback()" :class="['datetime-element', { 'mb-0': !columnLabel }, { 'is-focus': focus}]">
+    :state="isValid()" :invalid-feedback="getInvalidFeedback()" 
+    class="datetime-element" :class="[{ 'mb-0': !columnLabel }, { 'is-focus': focus}]">
```

Comment:
You can simplify to a single object literal: `:class="{ 'mb-0': !columnLabel, 'is-focus': focus }"`